src/components/Signin.jsx

```diff
@@ -0,0 +1,11 @@
+function Signin() {
+  return (
+    <form>
+      <label>Email: </label>
+      <input type="text" />
+      <button type="submit">Log in</button>
+    </form>
+  );
+}
```

Comment:
Please remove this for now. I get the idea behind it, but you won' be using email to log in. Took me a while to realise how to get to your articles!

Reply:
Remove the entire component?  Or just the email label for now?